plugins/autofrontlight.koplugin/_meta.lua

```diff
@@ -3,4 +3,5 @@ return {
     name = "autofrontlight",
     fullname = _("Auto frontlight"),
     description = _([[Automatically turns the frontlight on and off once brightness in the environment reaches a certain level.]]),
+    deprecated = "This plugin is unmaintained and will be removed soon.",
```

Comment:
I'd also overlooked the translation here.

Reply:
Good catch!

What happens when the plugin is deleted. Is the translation kept in cache for future uses or discarded?

If it is discarded we can keep a few generic messages as part of PluginLoader and use them on plugins.

Let me do a few tests ...